splunk_connect_for_snmp_poller/manager/poller_utilities.py

```diff
@@ -208,11 +212,11 @@ def automatic_realtime_task(
                     server_config,
                     splunk_indexes,
                 )
-                # touch inventory file after 2 min to trigger reloading of inventory with new walk data
-                schedule.every(2).minutes.do(
-                    refresh_inventory,
-                    inventory_file_path,
-                )
+                if not initial_walk:
```

Comment:
we schedule automatic_realtime_task with False in __start_realtime_scheduler_task
wouldn't we schedule new jobs every time it will run?

Reply:
The feature of forcing inventory refreshing is supposed to be called when new walk for device was conducted and in case sysDescr has changed we would like run the matching again. When we start the application automatic_realtime_task is called to schedule first walk for device and there is no need to force inventory reloading.